packages/target-ethers-v4/src/generation.ts

```diff
@@ -224,6 +226,27 @@ function generateInterfaceFunctionDescription(fn: FunctionDeclaration): string {
 `
 }
 
+function generateFunctionDocumentation(doc?: FunctionDocumentation): string {
```

Comment:
Here we will generate solidity types (like `uint256` for `@returns` and `@param`) for people using typescript... I am not sure if this won't confuse IDEs (typescript type annotation will clash).

What do you think about this problem? Can u paste here some generated commented for a real-world function? so we can see the output? Maybe we should omit type information in these comments entirely and just use human-readable part of docs? 🤔

Reply:
This is actually not generating types, but only comments. Here's a real-life comment:
```
    /**
     * Downgrades are not allowed, i.e. `_newVersion` should be higher than the currect colony version.
     * Upgrades a colony to a new Colony contract version `_newVersion`.
     * @param _newVersion The target version for the upgrade
     */
    upgrade(
      _newVersion: BigNumberish,
      overrides?: TransactionOverrides
    ): Promise<ContractTransaction>;
```

Not sure if this is what you were asking.

Reply:
Oh perfect. I get it now. I thought it's gonna have type annotation as well. This is great! Can u proceed with implementation for rest of the targets? 

When done I'll merge it to `kk/new-architecture` branch and release new beta packages.

Reply:
Yup, I'll continue tomorrow!